raspbian/install-raspbian.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+BOOTDISK=$1
+ROOTDISK=$2
+BOOTMNT=$3
+ROOTMNT=$4
+MNTDIR=/mnt
+IMGBOOT="imageboot"
+IMGROOT="imageroot"
+
+#Check if 5th argument was provided.
+#If not, check if the file already exist. If it doesnt exist, download the file. and then unzip it.
+#If the 5th argument is provided, unzip the file from that path.
+if [ -z "$5" ]; then
+	if [ ! -f raspbian_lite_latest ]; then
+		wget https://downloads.raspberrypi.org/raspbian_lite_latest
+	fi
+	unzip -o 'raspbian_lite_latest'
+else
+	unzip -o "$5"
+fi
+
+#Get the start location for root and boot partitions in MiB. 
+bootstart=$(sudo parted -sm *lite.img unit MiB print | grep 'fat32' | cut -f2 -d: | cut -f1 -d.)
+rootstart=$(sudo parted -sm *lite.img unit MiB print | grep 'ext4' | cut -f2 -d: | cut -f1 -d.)
+
+#Convert into bytes to get the offset.
+bootoffset=$(($bootstart*1048576))
+rootoffset=$(($rootstart*1048576))
+
+echo "imageroot $MNTDIR/$IMGROOT"
+echo "imageboot $MNTDIR/$IMGBOOT"
+echo "rootmnt $ROOTMNT"
+echo "bootmnt $BOOTMNT"
+echo "bootdisk $BOOTDISK"
+echo "rootdisk $ROOTDISK"
+
+umount $BOOTDISK
+umount $ROOTDISK
+
+rm -r $BOOTMNT; sudo mkdir $BOOTMNT
```

Comment:
Before this step, print at least a message asking the user if he wish to continue, deleting stuff is serious.

Reply:
Hey @patrickelectric and @jaxxzer, can you suggest what I can display to ask for the user's confirmation? I was thinking something along the lines of, "Continuing further will delete the root and boot mountpoints for the disk and the image; and create them again. Are you sure you want to continue?"
I feel this is too long but i dont know what would be a short way to explain what is actually going to happen and i dont know if it will even make sense to someone who doesnt know much about this. 
Please let me know.